### DISCUSSION TITLE
Verfiy command?

### DISCUSSION THREAD

**Discussion by jmerifjKriwe** [SukramJ/aiohomematic#864] [Q&A]
Is there a possibility to verify that a command has been successfully performed by an actor?
I have some actors which are a little bit "special" - I guess due to a not ideal connection. And sometimes they don't get a command. For example my outdoor lights. 
As a workaround I could send the command, wait 10 seconds, resend the command, wait another 10 seconds, resend again ... just to be sure. But I don't like this idea and it would bloat my automations.
So is there any kind of verification I can enable?

**Comment by HiranChaudhuri** [comment]
What you are asking for is already implemented on the lowest level in Homematic.
https://www.homematic-inside.de/faq/bidcos

> BidCoS® steht für Bidirectional Communication Standard. Dies ist ein Funkstandard, der speziell für die drahtlose Ansteuerung von Sensoren und Aktoren zur Hausautomation entwickelt wurde.

and further

> Die schnelle bidirektionale Kommunikation (gesendete Funksignale werden vom Empfänger bestätigt) erhöht die Funktionssicherheit und bildet die Grundlage für die umfangreichen Möglichkeiten der Fernsteuerung/-überwachung solcher Systeme.
